ckcp/openshift_dev_setup.sh

```diff
@@ -269,15 +291,19 @@ patches:
     --kubeconfig "$KUBECONFIG_KCP" \
     --kcp-org "$kcp_org" \
     --kcp-workspace "$kcp_workspace" \
-    --work-dir "$WORK_DIR"
+    --work-dir "$WORK_DIR" \
+    --kustomization "$GIT_URL/gitops/kcp/pac-manager?ref=$GIT_REF"
   KUBECONFIG_KCP="$WORK_DIR/credentials/kubeconfig/kcp/ckcp-ckcp.${ws_name}.${kcp_workspace}.kubeconfig"
 }
 
-install_compute() {
+install_pipeline_service() {
```

Comment:
This function does not seem quite generic enough yet. I think It should be split in 2.

`install_compute` that setups and deploys the compute, which are the generic steps that needs to be run in all cases, and `install_pipeline_service` which deploys pipelines-as-code, which we only need in our case.

If we wanted to go a step further, we could rename `install_pipeline_service` to `install_custom_compute`, put that in a `custom.sh` script. The script would be sourced if present, and the `install_custom_compute` function run.

Reply:
Great idea @Roming22. I'd like to propose a further change.
1. install_compute() --> It references [setup_compute.sh](./images/access-setup/content/bin/setup_compute.sh). And would generate access credentials for a new cluster so it can be managed by PAC. Also, it setups the $WORK_DIR by populating the Argo CD application source path to be referenced by the kustomization in $WORK_DIR (compute's). Although, in case of ckcp, it's not so generic. Because a kcp environment using ckcp with workload clusters wouldn't need this. Is this correct?   
2. install_custom_compute() (install_pipeline_service())  --> It references [install.sh](./images/cluster-setup/install.sh). It would deploy  Pipeline Service on the clusters as per the configuration in $WORK_DIR. For this, install_compute() is definitely a dependency.
3. install_pipelines_as_code() --> It references [setup.sh](./gitops/pac/setup.sh). For now, we install PAC controllers using the script in ckcp.